gordon/main.py

```diff
@@ -148,28 +124,21 @@ def _gather_runnable_plugins(plugins, debug):
     return plugins_to_run
```

Comment:
Can't comment on the lines directly above, but I _think_ the `elif` branch can be removed now, right?

Reply:
The elif checks if something implements runnable without explicitly declaring that it implements runnable.  I'm fine with dropping it!

Reply:
yeah I think I'd be fine w/ dropping it too.